apps/ios/LiveVideo/LiveVideo/Views/Stream/StreamToolbar.swift

```diff
@@ -12,7 +12,7 @@ struct StreamToolbar<Content>: View where Content: View {
     }
     
     var body: some View {
-        HStack(spacing: 10) {
+        HStack(spacing: 0) {
```

Comment:
Squeeze in a chat button for now. UX will need to figure out how we should add chat.